libafl/src/executors/forkserver.rs

```diff
@@ -761,10 +774,11 @@ impl<'a> ForkserverExecutorBuilder<'a, StdShMemProvider> {
     }
 
     #[must_use]
-    /// Place the input at this position, this is equivalent to putting "@@" here.
-    pub fn arg_input_file(mut self) -> Self {
```

Comment:
I'd add an `arg_input_file_std` with default filename (sorry that should have gone in the other PR)